spec/public-api.yaml

```diff
@@ -1367,8 +1375,7 @@ components:
           - ALGORITHM
         count:
           type: integer
-          minimum: null
```

Comment:
I made a fix earlier in the week, these are just plainly invalid swagger that is corrected

Reply:
Incidentally, to whom it may concern, this is a great website
https://editor.swagger.io/